indexer/packages/v4-protos/src/codegen/klyraprotocol/bridge/tx.ts

```
@@ -0,0 +1,551 @@
+/** MsgAcknowledgeBridges is the Msg/AcknowledgeBridges request type. */
+
+export interface MsgAcknowledgeBridgesSDKType {
+  /** The events to acknowledge. */
+  events: BridgeEventSDKType[];
+}
+/**
+ * MsgAcknowledgeBridgesResponse is the Msg/AcknowledgeBridgesResponse response
+ * type.
+ */
+
+export interface MsgAcknowledgeBridgesResponse {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)